app/src/unitTests/java/com/artemzin/qualitymatters/models/PicassoBitmapClientTest.java

```diff
@@ -0,0 +1,56 @@
+package com.artemzin.qualitymatters.models;
+
+import android.support.annotation.NonNull;
+import android.widget.ImageView;
+
+import com.artemzin.qualitymatters.QualityMattersRobolectricUnitTestRunner;
+import com.squareup.picasso.Picasso;
+import com.squareup.picasso.RequestCreator;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(QualityMattersRobolectricUnitTestRunner.class)
+public class PicassoBitmapClientTest {
+
+    @SuppressWarnings("NullableProblems") // Initialized in @Before.
+    @NonNull
```

Comment:
Well, idk idk, usually when I open some existing code that I haven't seen for some time I'd better see nullability annotations that not :)